yarn-project/circuits.js/src/tests/factories.ts

```diff
@@ -949,3 +951,18 @@ export function makeBaseRollupInputs(seed = 0): BaseRollupInputs {
 export function fr(n: number): Fr {
   return new Fr(BigInt(n));
 }
+
+/**
+ * Computes a valid address, partial address, and public key out of a private key.
+ * @param privateKey - A private encryption key (optional, will use a random one if not set).
+ * @returns A valid address, partial address, and public key.
+ */
+export async function makeAddressFromPrivateKey(privateKey?: Buffer) {
```

Comment:
I would expect this function to only return address. Maybe we can introduce AccountInfo interface and rename the function `getAccountInfo`? (I am open to any other naming)